### PR TITLE
Use the turbine-js-framework version, not the turbine-js-cli version …

### PR DESCRIPTION
**Pull request by janelletavares**
…in the version command and in the IR spec

### CODE REVIEW

File: packages/turbine-js-cli/bin/turbine-js-cli [hunk 1]
```diff
@@ -52,6 +52,7 @@ async function executeCommand() {
   } else if (args[0] === "clideploy") {
     // meroxa apps deploy step 3 (run users data app / create meroxa platform entities)
     //arg 0 command, arg 1 image, arg 2 path, args 3 appname, args 4, git sha, args 5, spec
+    const version = require(path.join(args[1] || process.cwd(), "package.json")).version;
```
Comment (jayjayjpg) [inline packages/turbine-js-cli/bin/turbine-js-cli:55]: If you like, you can define this line for setting the `version` variable at the top of the file where the previous `version` variable was set -- this way, you only have to write this statement once and you'll be able to reuse the avariable both in line 60 and on line 70

What do you think?

**Review by jayjayjpg — COMMENTED** [review]
This reads great! I left one more suggestion to save additional statements, but otherwise, this is already good to go. Thank you for the swift update! :sparkles:

File: packages/turbine-js-cli/bin/turbine-js-cli (hunk 1 above)
Comment (janelletavares) [reply packages/turbine-js-cli/bin/turbine-js-cli:55]: oooh ok.  i wasn't sure if the args[1] part would cause any panicking.

**Review by jayjayjpg — APPROVED** [review]
This reads great, thank you for the quick fix! :sparkles:

**Pull request merged**